itou/archive/management/commands/anonymize_jobseekers.py

```diff
@@ -87,6 +87,24 @@ def anonymized_jobapplication(obj):
     )
 
 
+def anonymized_approval(obj):
+    return AnonymizedApproval(
+        origin=obj.origin,
+        origin_company_kind=obj.origin_siae_kind,
+        origin_sender_kind=obj.origin_sender_kind,
+        origin_prescriber_organization_kind=obj.origin_prescriber_organization_kind,
+        start_at=get_year_month_or_none(obj.start_at),
+        end_at=get_year_month_or_none(obj.end_at),
+        had_eligibility_diagnosis=False if obj.eligibility_diagnosis_id is None else True,
```

Comment:
Tu peux caster en booléen directement
```suggestion
        had_eligibility_diagnosis=bool(obj.eligibility_diagnosis_id)
```